### PR TITLE
[MUI-551]: Create Navigation Component Backoffice

### PR DESCRIPTION
**Pull request by mr-teddy-bear**
# Description

Create Navigation Component Backoffice

## Jira ticket(optional)

[MUI-551](https://minterestfinance.atlassian.net/jira/software/projects/MUI/boards/7?assignee=62591503eee0a9006ab98b74&selectedIssue=MUI-551)

## Type of change

Please pick options that are relevant.

- [ ] Bug fix OR refactoring (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature or refactoring that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by marinakulsha** [comment]
Issues:
1. Active green Link:
AR: font-weight 600
ER: font-weight 400
2. the distance between title and bottom line according to design should be 10px

File: src/components/Navigation/Navlink.tsx [hunk 1]
```diff
@@ -7,27 +7,35 @@ import { Typography } from 'components';
...
+  ({
+    active,
+    backOfficeStyle,
+  }: {
+    active?: boolean;
+    backOfficeStyle?: boolean;
+  }) => ({
+    '&:hover': {
+      // todo move color in palette
+      backgroundColor: '#F0F1F2',
```
Comment (p1geon) [inline src/components/Navigation/Navlink.tsx:20]: should the color be moved to a pallet?

File: src/components/Navigation/Navlink.tsx [hunk 2]
```diff
@@ -7,27 +7,35 @@ import { Typography } from 'components';
...
-    backgroundColor: '#F0F1F2',
-  },
-  '&:disabled': {
-    opacity: 0.3,
-  },
-  // todo move color in palette
-  borderBottom: active ? '2px solid #00C7CF' : 'none',
-}));
+    borderBottom: active
+      ? '2px solid #00C7CF'
```
Comment (p1geon) [inline src/components/Navigation/Navlink.tsx:27]: this color exists in the palette

**Review by p1geon — APPROVED** [review]

File: src/components/Navigation/BackOfficeMenu.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+import { IDrawerProps } from './Drawer';
+import Navlink from './Navlink';
+
+const NavBar = styled('div')(() => ({
+  display: 'flex',
+  alignItems: 'center',
+}));
+
+const BackOfficeMenu: FC<
+  Omit<IDrawerProps, 'isOpen' | 'onClose' | 'ExtraComponent' | 'title'>
```
Comment (BerRossDolmat) [inline src/components/Navigation/BackOfficeMenu.tsx:14]: better to move to type, so will be easier to reuse

File: src/components/Navigation/Navlink.tsx [hunk 4]
```diff
@@ -52,12 +62,29 @@ const Navlink: React.FC<NavlinkT> = ({
...
+      onClick={onClickHandler}
+    >
       <NavlinkText
         sx={{
-          color: active
-            ? theme.palette.secondary.main
-            : theme.palette.action.disabled,
+          padding: backOfficeStyle ? '10px 24px' : '16px 24px',
+          fontWeight: backOfficeStyle ? 400 : active ? 600 : 400,
+          color:
```
Comment (BerRossDolmat) [inline src/components/Navigation/Navlink.tsx:75]: double ternary

**Review by BerRossDolmat — APPROVED** [review]

File: src/components/Navigation/Navlink.tsx (hunk 1 above)
Comment (gavrilin-remote) [reply src/components/Navigation/Navlink.tsx:20]: not now, a bit later****

**Pull request merged**